scripts/inactive_issue_checker.py

```diff
@@ -465,7 +466,11 @@ def unassign_issues(self, issues: List[Issue]) -> None:
                 )
 
 
-if __name__ == '__main__': # pragma: no cover
+def main() -> None:
+    """Main function for collecting all
+    inactive issues and deassign them.
```

Comment:
I think you can move some of the words here to the previous line.

Also, deassign --> deassigning